doc/src/build/move.md

```diff
@@ -767,13 +767,31 @@ At this point, however, the
 `sui-move` command does not support package publishing. In fact, it is
 not clear if it even makes sense to accommodate package publishing,
 which happens once per package creation, in the context of a unit
-testing framework. Instead, one can use a sample Sui wallet to
+testing framework. 
+
+Instead, one can use a sample Sui wallet to
 [publish](wallet.md#publish-packages) Move code and to
 [call](wallet.md#calling-move-code) it. See the
 [wallet documentation](wallet.md) for a description of how
 to publish the package we have [written](#writing-a-package) as as
 part of this tutorial.
 
+Another option is to use the [REST](rest-api.md) endpoints to [publish](rest-api.md#post-publish) 
```

Comment:
I think this whole section belongs to the REST API tutorial as the module.md document is more about developing Move code rather than a manual for the `sui-move` command.

I am not religious about it, though - perhaps @Clay-Mysten can weigh in here.

Reply:
I agree, the move.md file is focused upon smart contracts and already quite detailed. Not sure of the module.md file Adam speaks of (likely a typo).

Please move these contents to the Rest API doc (rest-api.md) in the same build/ directory.